src/VisualStudio/IntegrationTests/VisualBasic/BasicBuild.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class BasicBuild
+    {
+        private IntegrationHost _host;
+
+        public BasicBuild(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
```

Comment:
I'd disagree here: I think there's a beauty to the test classes being self contained. Otherwise we end up with insane inheritance hierarchies that only make what the code is doing _less_ clear. Duplicated code isn't bad; complexity is.

I am curious though why each test has to call .Initialize()...?